guidance/models/_model.py

```diff
@@ -1002,7 +1000,7 @@ def _repr_html_(self):
             clear_output(wait=True)
         return self._html()
 
-    def _current_prompt(self):
+    def _current_prompt(self) -> str:
```

Comment:
Other methods seem to annotate prompts as `bytes`, and the comment below says bytes -- should this return bytes or str?

Reply:
Actually, I think I should remove this. This annotation was a bit of an accident, before I realised there was a whole block of dead code

Reply:
Removed it.